docs/mongocli/command/mongocli-cloud-manager-logs-keyProviders-list.txt

```diff
@@ -0,0 +1,71 @@
+.. _mongocli-cloud-manager-logs-keyProviders-list:
+
+=============================================
+mongocli cloud-manager logs keyProviders list
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Lists all key provider configurations found in the encrypted log file.
```

Comment:
Add "audit", to make it read as follows:

... in the encrypted audit log file.

Reply:
done